src/main/webapp/app/detail-overview-list/components/no-data-component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'jhi-no-data',
+    template: `<span>-</span>`,
+    standalone: true,
+})
+export class NoDataComponent {}
```

Comment:
I think we use `-` for no data in more places in Artemis. Should we move this new component into a common package, so all other components can use it as well?

Reply:
Thanks, I was not aware that we have this solution at multiple places!

I moved the component to the shared folder and re-used it at places where the same solution was used for displaying no data